src/components/NavBar.js

```diff
@@ -1,15 +1,20 @@
 import React from "react"
+import {connect} from 'react-redux';
```

Comment:
Are we using Redux in this at all or can we remove the import and comments?

Reply:
that was left over from a failed attempt

Reply:
yeet